src/write_gcm.c

```diff
@@ -3348,6 +3349,9 @@ static int wg_transmit_unique_segment(const wg_context_t *ctx,
     }
 
     wg_log_json_message(ctx, "Server response:\n%s\n", response);
+    if (strncmp(response, "{}", 2) != 0) {
```

Comment:
That's an interesting point. Because it's supposed to come back with JSON, there won't be "somegarbage", but I can change it to be more precise if you care. (I think in practice what comes back right now is a newline after the {})